### ISSUE TITLE
Webpack-HMR - HTTPS

### ISSUE THREAD

**Issue by lukepolo** [JeffreyWay/laravel-mix#61]
We can start webpack-dev-server in https mode by adding `-https` but the files loading in always come from `http` , is this because of webpack or something you've setup in mix?

**Comment by QWp6t** [comment]
output is hard-coded here: https://github.com/JeffreyWay/laravel-mix/blob/master/src/Mix.js#L97

it's used here: https://github.com/JeffreyWay/laravel-mix/blob/master/setup/webpack.config.js#L65

So I think you should be able to add this to L66 of webpack config file:

```js
module.exports.output.publicPath = 'https://localhost:8080';
```

I'm not sure if that's what's causing your issue, but I think it might be.

**Comment by lukepolo** [comment]
Yah this probably should be configurable, i can look at making a PR later this week

**Comment by LG0012** [comment]
Hey lukepolo, do you have some issues with https certificates?

**Comment by lukepolo** [comment]
I dont have issues with the certificate it self, just how it references it

**Comment by LG0012** [comment]
Hmm, what about css reload? Maybe you know the solution how to join
browsersync css injection with HMR? :/

**Comment by lukepolo** [comment]
this issue has nothing todo with what your talking about , css reload does not work with HMR sadly (there is a way but dont think it will be merged in here) . And i have not dealt with browserync

**Comment by diego-vieira** [comment]
@lukepolo #446 did the trick for me

**Comment by JeffreyWay** [comment]
This was fixed in c74d71d1832946dbb5386dcf266378d9ab97e299
